ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -10,7 +10,7 @@ REHEARSAL_INDEX="brew.registry.redhat.io/rh-osbs-stage/iib:23576"
 REHEARSAL_PACKAGE="e2e-test-operator"
 REHEARSAL_CHANNEL="4.3"
 REHEARSAL_INSTALL_NAMESPACE="!create"
-REHEARSAL_TARGET_NAMESPACES="!install"
+REHEARSAL_TARGET_NAMESPACES="!all"
```

Comment:
I don't think you need to touch this: this triggers some special jobs to be rehearsed which is not desirable.

Reply:
Sounds good, fixed it.